bundles/org.openhab.io.openhabcloud/README.md

```diff
@@ -140,7 +140,7 @@ The additional parameter for these variants have the following meaning:
 - `title`: The title of the notification. Defaults to "openHAB" inside the Android and iOS apps.
 - `referenceId`: A user supplied id to both replace existing messages when pushed, and later remove messages with the `hideNotificationByReferenceId` actions.
 - `onClickAction`: The action to be performed when the user clicks on the notification. Specified using the [action syntax](#action-syntax).
-- `mediaAttachmentUrl`: The URL of the media attachment to be displayed with the notification. This can either be a fully qualified URL, prefixed with `http://` or `https://` and reachable by the client device, a relative path on the user's openHAB instance starting with `/`, or an image item with the format `item:MyImageItem`
+- `mediaAttachmentUrl`: The URL of the media attachment to be displayed with the notification. This can either be a fully qualified URL, prefixed with `http://` or `https://` and reachable by the client device, a relative path on the user's openHAB instance starting with `/`, or an image item with the format `item:MyImageItem`.  If the media attachment URL is not reachable by the client device (ie. the URL is not available on the public internet), use the `setImage` action to set the image item to the URL first then use `item:MyImageItem` as the media attachment URL. See [example](#examples) for more details.
```

Comment:
```suggestion
- `mediaAttachmentUrl`: The URL of the media attachment to be displayed with the notification. This can either be a fully qualified URL, prefixed with `http://` or `https://` and reachable by the client device, a relative path on the user's openHAB instance starting with `/`, or an image item with the format `item:MyImageItem`. If the media attachment URL is not reachable by the client device (ie. the URL is not available on the public internet), use the `setImage` action to set the image item to the URL first then use `item:MyImageItem` as the media attachment URL. See [example](#examples) for more details.
```